Exiled.Events/Patches/Events/Scp096/AddingTarget.cs

```diff
@@ -33,6 +35,12 @@ namespace Exiled.Events.Patches.Events.Scp096
     [HarmonyPatch(typeof(Scp096), nameof(Scp096.AddTarget))]
     internal static class AddingTarget
     {
+        // Nao convert to transpiler pls
+        private static bool Prefix(GameObject target)
+        {
+            return !target.IsNpc();
```

Comment:
Add a property in Npc for know if it's should trigger the SCP

Reply:
Why would you want that

Reply:
Because someone may decide to try and write a rudimentary AI to control bot players that actually play the game, in which case, the human roles should trigger 096 just like regular human players.

Reply:
makes sense ig

Reply:
added in 37447816e33576ea836a966826ff058817e40d66